### DISCUSSION TITLE
MacOS Ventura - Idea 2022.3 - password file doesn't work

### DISCUSSION THREAD

**Discussion by regal-wpetelski** [sadv1r/ansible-vault-editor-idea-plugin#193] [Q&A]
I've tried to use password file with plugin but it doesn't work - don't want to create issue but rather check if anyone else has same issue.
I've tried with ANSIBLE_VAULT_PASSWORD_FILE, ansible config, both plain text password and script with call to AWS SSM,
but with no luck - each time I see password prompt. Can someone confirm that it's possible? Thanks.

**Comment by sadv1r** [comment]
You can use a command like this for setting global environment variable on MacOS, but it will be lost after system restart, so you need to find how to save it after restart.

```sh
launchctl setenv ANSIBLE_VAULT_PASSWORD_FILE ~/passfile.txt
```

Another way is to use ansible.cfg instead of the environment variable.
